libr/anal/passmanager/bbpm.c

```diff
@@ -0,0 +1,4 @@
+#include <r_util.h>
+#include <r_passmanager/bbpm.h>
```

Comment:
thats wrong naming, r_passmanager is not a root module in r2land, so includename cant be like like this